cmd/internal.go

```diff
@@ -105,7 +115,7 @@ func (r *ParseCommand) ParsePipeline(assetPath string) error {
 
 	foundPipeline.WipeContentOfAssets()
 
-	js, err := json.Marshal(foundPipeline)
+	js, err := json.MarshalIndent(foundPipeline, "", "  ")
```

Comment:
I don't think we should do this, these are being passed onto the vscode extension, the smaller the payload the better

Reply:
you can always pipe this to `jq` to make it look nicer when working